grass7/imagery/i.zero2null/i.zero2null.html

```diff
@@ -8,6 +8,44 @@ <h2>DESCRIPTION</h2>
 in water bodies. These patches are removed and filled with neighboring
 cells.
 
+
+<h2>EXAMPLE</h2>
+
+The Sentinel-2 scene <tt>S2B_MSIL2A_20190724T103029_N0213_R108_T32ULA_20190724T130550,
+uuid: 0a7cb5ee-80d4-4d15-be19-0b3fdf40791f</tt> shows unexpected no-data pixels
+in lakes.
+
+<div class="code"><pre>
+# download S2 scene affected by no.data pixels within the scene
+i.sentinel.download settings=credentials.txt \
+  uuid=0a7cb5ee-80d4-4d15-be19-0b3fdf40791f output=test_s2_scene
```

Comment:
I think I provided the wrong UUID - maybe to avoid confusion, please use the identifier S2B_MSIL2A_20190627T104029_N0212_R008_T32ULC_20190627T135004

Reply:
yes, the example is now functional